src/resources/settings.yml

```diff
@@ -20,14 +20,23 @@ reverse_proxy:
       # Settings pertaining to the ACME DNS challenge type.
       acme_dns_challenge:
         # Whether to use ACME DNS challenge (rather than HTTP challenge) to acquire certificates.
-        enable: true
+        enable: false
 
         # AWS access key with permissions to manipulate Route53 records for DNS challenge.
         access_key_id: CHANGE ME
 
         # AWS access key with permissions to manipulate Route53 records for DNS challenge.
         secret_access_key: CHANGE ME
 
+    # Settings pertaining to monitoring.
+    monitoring:
+      # Whether to enable the montoring stack.
+      # This takes the `caddy` logs and exports them to a elasticsearch server.
+      enabled: true
+
+      # Elasticsearch server URL.
+      elasticsearch_url: "elasticsearch:9200"
```

Comment:
It's confusing calling this url and not having http/https.
In the logstash configs, this gets mapped through to `hosts`, so let's just call it `elasticsearch_host` and update the comment to be something like

```
The Elasticsearch API endpoint to send logs to (defaults to `elasticsearch:9200`).
```
You default it later on in one of the low level files, you need to capture that otherwise users get confused.